src/pages/Metrics/Metrics.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react';
+import { PageHeaderWrapper } from '@ant-design/pro-layout';
+import { Empty, Button, Card } from 'antd';
+import { history } from 'umi';
+import { stringify } from 'qs';
+
+const Metrics: React.FC<{}> = () => {
+  const GLOBAL_ADMIN_SETTING_GRAFANA_URL =
+    localStorage.getItem('GLOBAL_ADMIN_SETTING_GRAFANA_URL') || undefined;
+  const [showMetricsDashboard] = useState(GLOBAL_ADMIN_SETTING_GRAFANA_URL);
```

Comment:
1. showMetrics is fine;
2. useState(Boolean(localStorage.getItem('GLOBAL_ADMIN_SETTING_GRAFANA_URL')))

Reply:
or `const grafanaURL = xxx`